scripts/req.http

```diff
@@ -0,0 +1,18 @@
+# For a quick start check out our HTTP Requests collection (Tools|HTTP Client|Open HTTP Requests Collection) or
+# paste cURL into the file and request will be converted to HTTP Request format.
+#
+# Following HTTP Request Live Templates are available:
+# * 'gtrp' and 'gtr' create a GET request with or without query parameters;
+# * 'ptr' and 'ptrp' create a POST request with a simple or parameter-like body;
+# * 'mptr' and 'fptr' create a POST request to submit a form with a text or file field (multipart/form-data);
+
+GET http://localhost:8080/postWithComments
+Accept: application/json
+
+###
+
+
+GET http://localhost:8080/postWithCommentsInXml
+Accept: application/json
+
+###
```

Comment:
Anyway, thanks for your submission and apologize for the delay in code-review.
Happy Coding!